models/labels/funds/labels_funds_ethereum.sql

```diff
@@ -0,0 +1,25 @@
+{{config(alias='funds_ethereum')}}
+
+SELECT blockchain, address, name, category, contributor, source, created_at, updated_at
+FROM (VALUES
+     (array('ethereum'),'0x2B1Ad6184a6B0fac06bD225ed37C2AbC04415fF4', 'a16z', 'funds', 'soispoke', 'static', timestamp('2022-10-03'), now())
```

Comment:
intended to be september 3rd?

Reply:
Yes definitely 😅 I modified this in a commit